system-int-test/src/cmr/system_int_test/data2/umm_spec_subscription.clj

```diff
@@ -38,7 +38,11 @@
   ([attribs]
    (subscription attribs latest-umm-sub-verison))
   ([attribs version]
-   (umm-sub/map->UMM-Sub (merge (sample-umm-subscription version) attribs))))
+   (let [sample-subscription (merge (sample-umm-subscription version) attribs)
```

Comment:
I made a similar change in my PR and it is merged now. You can remove this change and just pull in my changes.